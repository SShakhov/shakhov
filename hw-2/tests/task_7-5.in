1If 2you 3wanna 4be 5my 6friend, 7put 8my 9dick 10in 11your 12hand
