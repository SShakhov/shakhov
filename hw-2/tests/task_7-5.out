1If 2you 3wanna 4be 5my 6friend, 7put 8my 9dick 10in 11your 12hand
1. All numbers in front in the same order
123456789101112If you wanna be my friend, put my dick in your hand
